src/utils/RacingCarUtils.js

```diff
@@ -0,0 +1,53 @@
+import {Console, Random} from '@woowacourse/mission-utils';
+import RacingCar from "../RacingCar.js";
+import ValidationUtils from "./ValidationUtils.js";
+import ErrorCode from "../datas/ErrorCode.js";
+
+class RacingCarUtils {
+    static findWinners(carList) {
+        if (carList.length <= 0) throw new Error(ErrorCode.CAR_NOT_ENOUGH)
+
+        carList.sort((a, b) => b.getDistance() - a.getDistance())
+
+        const maxDist = carList[0].getDistance();
```

Comment:
```suggestion
        const maxDist = Math.max(...carList.map((car) => car.getDistance()));
```
`Math.max`를 사용해서 간략하게 작성하는건 어떨까요??

Reply:
저도 코드리뷰하면서 이런 방법이 있었는데 왜 이렇게 어렵게 생각했을까 했네요 ㅠㅠ
자바나 c++에서 항상 sort를 활용하던 습관때문인 것 같습니다